supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc

```diff
@@ -396,6 +396,17 @@ An Ansible task is a set of instructions to achieve a state defined, in its broa
 
 *See also*: xref:app[app]
 
+[discrete]
+[[application-stream]]
+==== image:images/yes.png[yes] Application Stream (noun)
+*Description*: _Application Streams_ are multiple versions of RHEL user-space components that are delivered and updated more frequently than the core operating system packages. Application Streams can be packaged as RPM packages, modules, or Software Collections. Do not confuse with "AppStream", which is the repository through which Application Streams are distributed. Note that the AppStream repository also contains components other than Application Streams.
```

Comment:
```suggestion
*Description*: _Application Streams_ are multiple versions of Red Hat Enterprise Linux user-space components that are delivered and updated more frequently than the core operating system packages. Application Streams can be packaged as RPM packages, modules, or Software Collections. Do not confuse with "AppStream", which is the repository through which Application Streams are distributed. Note that the AppStream repository also contains components other than Application Streams.
```
- Should RHEL be spelled out here?

Reply:
The SSG spells out "Red Hat Enterprise Linux throughout the glossary except in a few niche cases. Agree that the acronym should be spelled out. Ditto other cases in this PR.